src/pageObject/LayoutPage.ts

```diff
@@ -0,0 +1,25 @@
+import { Page, expect } from '@playwright/test';
```

Comment:
Better to move this verification to the test level and use strict ones whenever it possible